rudder-web/src/main/scala/com/normation/rudder/web/components/RuleEditForm.scala

```diff
@@ -1194,7 +1204,12 @@ class RuleEditForm(
                "sPaginationType": "full_numbers",
                "bJQueryUI": true,
                "bStateSave": true,
-               "sCookiePrefix": "Rudder_DataTables_",
+                    "fnStateSave": function (oSettings, oData) {
+                      localStorage.setItem( 'DataTables_reportsGrid', JSON.stringify(oData) );
+                    },
+                    "fnStateLoad": function (oSettings) {
+                      return JSON.parse( localStorage.getItem('DataTables_changeHistory') );
```

Comment:
DataTables_reportsGrid no ? And I'm not sur it's really the one we want since it's already used in line 812 of RuleEditForm above (I can't find my way on all the table :/)